### ISSUE TITLE
require('graphql-directive-auth') does not work as intended

### ISSUE THREAD

**Issue by tcerdaITBA** [graphql-community/graphql-directive-auth#16]
Hello!

I have installed the dependency via `npm install graphql-directive-auth` and in order to use it I must first retrieve the `default` property from `require('graphql-directive-auth')` like so:

```js
const AuthDirective = require('graphql-directive-auth').default;
```

Which is not how it is specified in the documentation. This does not happen when using the `import` keyword:

```js
import AuthDirectives from 'graphql-directive-auth';
```

 I believe this has something to do with TypeScripts module bundling.

**Comment by chanlito** [comment]
@tcerdaITBA yeah, I think this lib should put `"module": "commonjs"` to tsconfig.json

**Comment by FluorescentHallucinogen** [comment]
Is this issue fixed in 4979f42defc32a6c0aea7486b908ec24f56a5147?
